lib/ui/views/home/widgets/restaurant_card/restaurant_card.dart

```diff
@@ -17,11 +19,10 @@ class RestaurantCard extends StatelessWidget {
     return ViewModelBuilder<RestaurantCardModel>.reactive(
```

Comment:
should this be reactive? the restaurant card data is pre-loaded before it can render the UI, so it should be nonReactive. it shouldn't be re-rendering loads and consuming battery power unnecessarily